assets/fonts/WickedQueen.fnt

```diff
@@ -0,0 +1,90 @@
+info face="Wicked Queen BB" size=32 bold=0 italic=0 charset="" unicode=0 stretchH=100 smooth=1 aa=1 padding=1,1,1,1 spacing=-2,-2
```

Comment:
Hi @dacharya64 ! May I ask where did you create this `size=32` `.fnt` file for Wicked Queen? I am looking into having a larger size of this font within the game, and your help would be much appreciated as the author of this PR!